modules/ROOT/pages/passing-additional-arguments-to-the-jvm-to-control-mule.adoc

```diff
@@ -0,0 +1,21 @@
+= Passing Additional Arguments to the JVM to Control Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, esb, deploy, configure, jvm arguments, system properties, properties
+
+//The Wrapper provides several properties you can set as described in the article on http://wrapper.tanukisoftware.org/doc/english/properties.html[Java Service Wrappers].
```

Comment:
Since we don't link out to 3rd party sites and we try to avoid tech debt in topics, we should prob remove this commented-out part. That said, is there any useful info we should think about providing in our docs?